### ISSUE TITLE
Postgres container fails when upgrading from before to after #638 fix

### ISSUE THREAD

**Issue by blynchNDI** [nditech/apollo#639]
@takinbo and @dodumosu - The fix in pull request #638 does indeed seem to solve the problem with the upstream issue that prevents a fresh Apollo spin up. A new issue though is that it is not possible to upgrade an Apollo instance from before this commit to after it, due to what appears to be an issue with the old container running Postgres 10 and the new one running version 12.1. The Postgres container fails to start with the following error:
```
PostgreSQL Database directory appears to contain a database; Skipping initialization

2019-11-27 17:57:18.628 UTC [1] FATAL:  database files are incompatible with server
2019-11-27 17:57:18.628 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 10, which is not compatible with this version 12.1 (Debian 12.1-1.pgdg100+1).
```

**Comment by takinbo** [comment]
Yes, this is indeed an issue that I also observed. The fix we should work on is an upgrade script to upgrade any existing databases to the current version.

**Comment by takinbo** [comment]
[this repository](https://github.com/timbaobjects/apollo_postgres_upgrade) has been created with instructions on how to upgrade older data volumes with PostgreSQL 10 to the latest PostgreSQL 12

**Comment by takinbo** [comment]
@blynchNDI I've done some local testing but any live testing with feedback would be appreciated.

**Comment by blynchNDI** [comment]
Just tried out the fix on a few sites, and all seemed to upgrade without issue
